docs/csharp/programming-guide/namespaces/index.md

```diff
@@ -39,16 +39,16 @@ Namespaces are heavily used in C# programming in two ways. First, the .NET Frame
   
 -   The `using directive` obviates the requirement to specify the name of the namespace for every class.  
   
--   The `global` namespace is the "root" namespace: `global::System` will always refer to the .NET Framework namespace `System`.  
+-   The `global` prefix specifies that a namespace is fully qualified, and it begins at the "root" namespace: `global::System` will always refer to the .NET Framework namespace `System`.  
```

Comment:
I think the explanation at the global keyword page is a bit clearer. What if we added that to the beginning of this sentence?
The global contextual keyword refers to the global namespace, which is the default namespace for any C# program and is otherwise unnamed. 

`global::System` will always refer to the .NET Framework namespace `System`.  -> `global::System` always refers to the namespace \<xref:System> in .NET.  

And then add: For more information, see [How to: Use the Global Namespace Alias](how-to-use-the-global-namespace-alias.md).

Reply:
I finally realized what I've been struggling with:  All our explanations use `global::System` when we want to explain the `global` contextual keyword. Somehow we need to show that `global::Microsoft.Win32` always refers to the `Microsoft.Win32` namespace.